composer.json

```diff
@@ -46,7 +46,54 @@
             "Neos.Kickstart-20161124230102",
             "Neos.Kickstarter-20161125110814",
             "Neos.Flow-20161125124112",
-            "TYPO3.FluidAdaptor-20161130112935"
+            "TYPO3.FluidAdaptor-20161130112935",
```

Comment:
Actually we could remove the whole `applied-flow-migrations` section since core migrations are only applied to `neos/*` packages by default.

Reply:
Yeah, but that's the result if you run `./flow core:migrate <your-package>` as it is described in the update docs. So those will come back again if you migrate for the next version

Reply:
I know ;)
But nobody will run them, so it's not a problem *g

In earlier versions, Flow applied core migrations to _all_ Flow packages, but since that's not the case any more it's not an issue.
I'll take care when testing this change with Flow 5.0 later today!